src/csv_to_TM.py

```diff
@@ -0,0 +1,39 @@
+import csv
+import PYSHP
+from PYSHP import Writer
+
+with open('intervallen.csv', newline='') as csvfile:
+    data = csv.reader(csvfile, delimiter=';', quotechar='"')
+
+    for row in data:
+       meeuw = row[0] 
+       start = row[1].split('/')
+       einde = row[2].split('/')
+       eerste = start[2]+'-'+start[1].rjust(2, '0')+'-'+start[0].rjust(2, '0')
+       print(eerste)
+       laatste = einde[2]+'-'+einde[1].rjust(2, '0')+'-'+einde[0].rjust(2, '0')
+       
+       bestandsnaam = 'MeeuwenTM/'+ meeuw + '.gpx'             
+       file = open(bestandsnaam, 'w')
+       file.write("""<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
waarom werk je hier niet binnen een `ẁith` context? Dan hoef je die end closure niet te doen en ben je zeker dat goed aflsuit.

Reply:
Om eerlijk te zijn, heb ik daar niet echt bij stil gestaan. Het is voor mij persoonlijk al 3 jaar geleden dat ik nog een programmeer vak had. Ik zal het eens bekijken, hoe ik juist met die with context moet werken.

Reply:
Geen probleem, je doet het vr het openen van de originele file en is dus volledig analoog.